production/grafanacloud-install.sh

```diff
@@ -71,8 +71,30 @@ main() {
       ;;
   esac
 
-  log '--- Retrieving config and placing in /etc/grafana-agent.yaml'
-  retrieve_config | sudo tee /etc/grafana-agent.yaml
+  if [ -f /etc/grafana-agent.yaml ]; then
```

Comment:
This looks like the first use of needing user input in this install script. Maybe another option would be to abort if the file already exists and require a `--overwrite-config` option (or something similar)? That would prevent automated uses of this script from hanging waiting for user input.

Reply:
This is a great point. This script is a bit of a special case: we don't expect it to be involved in automation and only plan for it being used for a manual walk through, but I still think there's value in supporting flags here.